packages/cli/src/commands/hydrogen/env/list.ts

```diff
@@ -88,36 +91,50 @@ export async function listEnvironments({path, shop: flagShop}: Flags) {
   );
   storefront.environments.push(previewEnvironment[0]!);
 
-  const rows = storefront.environments.map(({branch, name, url, type}) => {
+  outputNewline();
```

Comment:
This used to output `undefined` in my terminal... glad it's working now 😅

Reply:
cc @matteodepalo have you heard about this magic before? 😅 

Maybe [its implementation](https://github.com/Shopify/cli/blob/4746f68fa45ae814af95c1a61bbcd6d6be975440/packages/cli-kit/src/public/node/output.ts#L312-L314) should be `console.log('')` instead of `console.log()`? -- We use the former directly in some places in this CLI to avoid printing `undefined`.

Reply:
I haven't used this function before so I'm not sure 🤔 Could we not print `\n`? Or would that cause double line? I think `console.log('')` makes sense though.